lib/stagehand/staging/model.rb

```diff
@@ -3,6 +3,10 @@ module Staging
     module Model
       extend ActiveSupport::Concern
 
+      included do
+        Stagehand::Configuration.staging_model_tables << table_name
```

Comment:
This has the potential to add duplicates if the model is reloaded dynamically in development mode.

Reply:
Resolved by using `Set`.